tests/core/sys/windows/test_clipboard.odin

```diff
@@ -0,0 +1,52 @@
+//+build windows
+package test_core_sys_windows
+
+import "core:testing"
+import win32 "core:sys/windows"
+import "base:runtime"
+import "core:strings"
+import "core:mem"
+
+read_from_clipboard :: proc(wnd_handle: win32.HWND, allocator: runtime.Allocator) -> (result: string)
+{
+	if !win32.IsClipboardFormatAvailable(win32.CF_TEXT) do return;
+	
+		if !win32.OpenClipboard(wnd_handle) do return;
+		defer win32.CloseClipboard();
+		
+		clipboard_data := win32.GetClipboardData(win32.CF_TEXT); 
+		if clipboard_data != nil {
+			if cstr := cstring(win32.GlobalLock(win32.HGLOBAL(clipboard_data))); cstr != nil {
+				result = strings.clone_from_cstring(cstr, allocator);
+			}
+			win32.GlobalUnlock(win32.HGLOBAL(clipboard_data));
+		}
+	return;
+}
+
+write_to_clipboard :: proc(wnd_handle: win32.HWND, text: string)
```

Comment:
You could possibly also make these procedures return `win32.BOOL` and go with win32.OpenClipboard(wnd_handle) or_return`, and so on.